packages/react-dom/src/events/ReactDOMEventListener.js

```diff
@@ -203,6 +192,29 @@ export function trapEventForPluginEventSystem(
   } else {
     fbListener = addEventBubbleListener(container, rawEventName, listener);
   }
+  // If we have an fbListener, then use that.
+  // We'll only have one if we use the forked
+  // EventListener-www module in FB builds.
+  return fbListener || listener;
+}
+
+export function removeTrappedPassiveEventListener(
+  document: Document,
+  topLevelType: string,
+  listener: any => void,
+) {
+  if (listener.remove != null) {
+    listener.remove();
+  } else {
+    if (passiveBrowserEventsSupported) {
+      document.removeEventListener(topLevelType, listener, {
+        capture: true,
+        passive: true,
+      });
+    } else {
+      document.removeEventListener(topLevelType, listener, true);
```

Comment:
This isn't going through the forked indirection which isn't the right layering. This whole function should all be moved to EventListener.js. That way it can be properly forked and you can statically choose which function to call.